common/visual-test.cpp

```diff
@@ -250,3 +250,11 @@ bool VisualTest::CompareImageFile(const std::string fileName1, const std::string
 
   return passed;
 }
+
+void VisualTest::EmitTouchEvent( TouchPoint& touchPoint )
```

Comment:
The function name doesn't match what you're actually doing. If you do need to submit two events then do this from the demo code but keep this correct at least.

Or rename the function.

Reply:
I think you need to tell me then what I'm doing because so far I though I was emitting touch events at a specific screen location.